app/components/timesheet-actions.js

```diff
@@ -1,22 +1,32 @@
 import Component from '@glimmer/component';
-import { task } from 'ember-concurrency';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import constants from 'frontend-timekeeper/constants';
+const { TIMESHEET_STATUSES } = constants;
 
 export default class TimesheetActionsComponent extends Component {
-  get canSubmitHolidays() {
-    return !this.args.timesheet
-      || this.args.timesheet.isDraft;
-  }
+  @tracked absenceComplete = false;
+  @tracked timesheetComplete = false;
```

Comment:
The component doesn´t take the current timesheet status into account

Reply:
Whoops, you're right, fixed in [f9cc580](https://github.com/redpencilio/frontend-timekeeper/pull/4/commits/f9cc5807b48c48f7beac0c1d941acddad31b2bbe)